### PR TITLE
Add logic to factories and seeders for Donor and Animal...

### PR DESCRIPTION
**Pull request by WaldemarGoszka**
...add Sizes enum, change faker-locale to PL in app configuration

### CODE REVIEW

File: app/Enums/Sizes.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+namespace App\Enums;
+
+/*
+ * This is mapped to the "species" database table
```
Comment (Guferos) [inline app/Enums/Sizes.php:5]: Can you change the word "species" to "sizes" please?

**Review by Guferos — CHANGES_REQUESTED** [review]
All good. Just needs changing "species" to "sizes" in the comment. Could you please make the update and I will merge.

File: app/Enums/Sizes.php (hunk 1 above)
Comment (WaldemarGoszka) [reply app/Enums/Sizes.php]: Yes I can.

Comment (WaldemarGoszka) [reply app/Enums/Sizes.php]: Is already done

**Pull request merged**